docs/content/user-guides/cert-manager.md

```diff
@@ -0,0 +1,155 @@
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: Issuer
+    metadata:
+      name: acme
+
+    spec:
+      acme:
+        # Production server is on https://acme-v02.api.letsencrypt.org/directory
+        # Use staging by default.
+        server: https://acme-staging-v02.api.letsencrypt.org/directory
+        privateKeySecretRef:
+          name: acme
+        solvers:
+          - http01:
+              ingress:
+                ingressClassName: traefik
+    ```
+
+!!! example "Certificate with this Issuer"
+
+    ```yaml
+    apiVersion: cert-manager.io/v1
+    kind: Certificate
+    metadata:
+      name: whoami
+      namespace: traefik
+    spec:
+      secretName: domain-tls        # <===  Name of secret where the generated certificate will be stored
+      dnsNames:
+        - "domain.example.com"
+      issuerRef:
+        name: acme
+        kind: Issuer
+    ```
+
+!!! example "Route with this Certificate"
+
+    ```yaml tab="Ingress"
+    apiVersion: networking.k8s.io/v1
+    kind: Ingress
+    metadata:
+      name: domain
+      annotations:
+        traefik.ingress.kubernetes.io/router.entrypoints: websecure
+
+    spec:
+      rules:
+      - host: domain.example.com
+        http:
+          paths:
+          - path: /
+            pathType: Exact
+            backend:
+              service:
+                name:  domain-service
+                port:
+                  number: 80
+      tls:
+      - secretName: domain-tls # <=== Use the name defined in Certificate resource
+    ```
+
+    ```yaml tab="IngressRoute"
+    apiVersion: traefik.io/v1alpha1
+    kind: IngressRoute
+    metadata:
+      name: domain
+
+    spec:
+      entryPoints:
+        - websecure
+
+      routes:
+      - match: Host(`domain.example.com`)
+        kind: Rule
+        services:
+        - name: domain-service
+          port: 80
+      tls:
+        secretName: domain-tls    # <=== Use the name defined in Certificate resource
+    ```
+
+    ```yaml tab="HTTPRoute"
+    ---
+    apiVersion: gateway.networking.k8s.io/v1
+    kind: Gateway
+    metadata:
+      name: domain-gateway
+    spec:
+      gatewayClassName: traefik
+      listeners:
+        - name: websecure
+          port: 8443
+          protocol: HTTPS
+          hostname: domain.example.com
+          tls:
+            certificateRefs:
+              - name: domain-tls  # <==== Use the name defined in Certificate resource
+    ---
+    apiVersion: gateway.networking.k8s.io/v1
+    kind: HTTPRoute
+    metadata:
+      name: domain
+    spec:
+      parentRefs:
+        - name: domain-gateway
+      hostnames:
+        - domain.example.com
+      rules:
+        - matches:
+            - path:
+                type: Exact
+                value: /
+
+          backendRefs:
+            - name: domain-service
+              port: 80
+              weight: 1
+    ```
+
+## Troubleshooting
+
+There are multiples sources available to investigate when using Cert Manager:
```

Comment:
Here I am wondering if we could add some link to the corresponding section in the Cert-Manager documentation.

Reply:
I found one and added it to this document.